diff --git a/CMakeLists.txt b/CMakeLists.txt
index bdd8370..1b3b2f7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,7 +34,7 @@ find_package(gz_utils_vendor REQUIRED)
 # if we are users are building from source (determined by the
 # GZ_BUILD_FROM_SOURCE environment variable) or if explicitly told to do so
 # by the GZ_RELAX_VERSION_MATCH environment variable.
-set(VERSION_MATCH "EXACT")
+set(VERSION_MATCH "")
 if(NOT $ENV{GZ_BUILD_FROM_SOURCE} STREQUAL "")
   set(VERSION_MATCH "")
 endif()
